Engine/SpellFxRenderer.cpp

```diff
@@ -193,7 +193,7 @@ void SpellFX_Billboard::_47829F_sphere_particle(
 
         uNumVertices = 3;
         if (SpellFXViewTransform() && SpellFXViewClip()) {
-            if (SpellFXProject()) render->_4A4CC9_AddSomeBillboard(this, diffuse);
+//            if (SpellFXProject()) render->_4A4CC9_AddSomeBillboard(this, diffuse);
```

Comment:
Is this intentional?

Reply:
No, this was wrongly commented out effect due to VS bug I mentioned in discord. It was fixed in latest commit.